src/middlewared/middlewared/api/v26_04_0/smb.py

```diff
@@ -162,6 +163,10 @@ class SmbServiceEntry(BaseModel):
     aapl_extensions: bool
     """ Enable support for SMB2/3 AAPL protocol extensions. This setting makes the TrueNAS server advertise support \
     for Apple protocol extensions as a MacOS server. Enabling this is required for Time Machine support. """
+    search_protocols: list[SMBSearchType] | None
```

Comment:
In which case would it be `None` and how would it be different from an empty list?

Reply:
Good point. Don't need to accept None here.